deployment/docker/vmanomaly/vmanomaly-vmalert-guide/README.md

```diff
@@ -0,0 +1,23 @@
+# Docker Compose file for "Getting Started with vmanomaly" guide
+
+Please read the "Getting Started with vmanomaly" guide first - [https://docs.victoriametrics.com/anomaly-detection/guides/guide-vmanomaly-vmalert.html](https://docs.victoriametrics.com/anomaly-detection/guides/guide-vmanomaly-vmalert.html)
+
+To make this Docker compose file work, you MUST replace the content of [vmanomaly_license.txt](https://github.com/VictoriaMetrics/VictoriaMetrics/tree/master/deployment/docker/vmanomaly/vmanomaly-vmalert-guide/vmanomaly_license.txt) with valid license.
+
+You can issue the [trial license here](https://victoriametrics.com/products/enterprise/trial/)
+
+
+## How to run 
+
+1. Replace content of `vmanomaly_license.txt` with your license
+1. Run
+
+```sh 
+docker compose up -d  
+```
+1. Open Grafana on  `http://127.0.0.1:3000/`
+```sh
+open http://127.0.0.1:3000/
+```
+
+If you see not date, wait for a couple of minutes
```

Comment:
a typo?
`If you see not date, wait for a couple of minutes` -> something like `If you see no data shown on a graph, wait for a couple of minutes`